### DISCUSSION TITLE
Terraforming Integration Suite Connectors

### DISCUSSION THREAD

**Discussion by cnbscorp** [SAP/terraform-provider-btp#621] [Ideas]
Hi all,
are there any plans to extend this provider to manage Integration Suite resources in the future? 
Specifically, I am interested if it will be possible to, lets say, create an Integration Suite subscription and terraform corresponding connectors e.g. an open connector instance of a specific type? 
Thanks a lot and great work with this provider, it really adds value to the BTP :)

**Comment by lechnerc77** [comment]
Hi @cnbscorp 

thanks for the  kind feedback , great to hear that!

Concerning the integration Suite: we do not have any plans to extend to scope in the direction of the Integration Suite. The central question is how to structure these topics (integrate in provider, create new provider) and how it fits in the architecture of the provider. 
However it would be interesting to learn more about the integration suite scenarios that you can think about especially the different lifecycles (setup of integration suite, configuration, deployment of flows).

If you are interested in a discussion, feel free to ping me via christian.lechner@sap.com

Best,
Christian

**Comment by lechnerc77** [comment]
@cnbscorp Thanks for the input via email. I created a feature request on this topic here: #625 
Further discussion should be placed in the issue, so I am locking this discussion to have "one source" for input on this topic
